package.json

```
@@ -19,17 +20,21 @@
     "curves"
   ],
   "scripts": {
-    "rollup": "run-s rollup:docs rollup:dist",
-    "rollup:base": "rollup ./lib/bezier.js --format esm --file ",
-    "rollup:docs": "npm run rollup:base -- ./docs/js/bezier.js",
-    "rollup:dist": "npm run rollup:base -- ./dist/bezier.js",
+    "build": "run-s build:docs build:dist build:cjs",
+    "build:base": "rollup ./lib/bezier.js --format esm --file ",
+    "build:docs": "npm run build:base -- ./docs/js/bezier.js",
+    "build:dist": "npm run build:base -- ./dist/bezier.js",
+    "build:cjs": "babel --plugins @babel/plugin-transform-modules-commonjs lib --out-dir dist/cjs",
```

we already have a single-file bundle in `/dist` at this point: let's run the conversion on that instead of on the `lib` directory. No reason to make this five separate files.
```suggestion
    "build:cjs": "babel --plugins @babel/plugin-transform-modules-commonjs dist/bezier.js --out-file dist/bezier.common.js",
```

---

@Pomax AFAIK this would create a bundle including dependencies, which is not the intention. shipping a package export as a bundle prevents the packer consumer from deduping dependencies at the app level

---

except this library _has no dependencies_

---

ok sure